src/panels/developer-tools/debug/developer-tools-debug.ts

```diff
@@ -21,10 +21,11 @@ class HaPanelDevDebug extends SubscribeMixin(LitElement) {
           class="form"
         >
           <div class="card-content">
-                  <ha-debug-connection-row
-                    .hass=${this.hass}
-                    .narrow=${this.narrow}
-                  ></ha-debug-connection-row>
+            <ha-debug-connection-row
+              .hass=${this.hass}
+              .narrow=${this.narrow}
+            ></ha-debug-connection-row>
+          </div>
```

Comment:
It's copy paste from the profile settings. The div should be closed before ha-debug-connection-row for the display to look ok. Card content class exists and adds padding and the form class can be removed.